versions/versions.gradle

```diff
@@ -6,8 +6,8 @@
 // than the consumer applications.
 ext.CONFIG.versions = [
     // Java compatibility target version
-    java      : '1.7',
-    kotlin    : '1.4.10',
+    java      : '11',
+    kotlin    : '1.4.31',
```

Comment:
should we already update this to 1.5.x?

Reply:
not all projects are updated, SDKUtils is still using v1.3.50

Reply:
yes, I think we can update this
and let other projects to use old one manually if necessary.

Reply:
Updated to v1.5.21